examples/kepler-integration/src/components/export-video/export-video-panel-settings.js

```diff
@@ -57,12 +57,58 @@ const InputGrid = styled.div`
   grid-row-gap: ${DEFAULT_ROW_GAP};
 `;
 
+/**
+ * Used to convert durationMs (inherited from ExportVideoPanelContainer) to hh:mm:ss
+ * @param {number} durationMs duration of animation in milliseconds
+ * @returns {string} time in format hh:mm:ss
+ */
+function msConversion(durationMs) {
+  let seconds = parseInt((durationMs / 1000) % 60, 10);
+  let minutes = parseInt((durationMs / (1000 * 60)) % 60, 10);
+  let hours = parseInt((durationMs / (1000 * 60 * 60)) % 24, 10);
+
+  hours = hours < 10 ? `0${hours}` : hours;
+  minutes = minutes < 10 ? `0${minutes}` : minutes;
+  seconds = seconds < 10 ? `0${seconds}` : seconds;
+
+  return `${hours}:${minutes}:${seconds}`;
+}
+
+/**
+ * Estimates file size of resulting animation
+ * @param {number} frameRate frame rate of animation (set by developer)
+ * @param {array} resolution [width, height] of animation
+ * @param {number} durationMs duration of animation (set by developer)
+ * @param {string} mediaType 'GIF', 'WEBM', etc.
+ * @returns {string} size in MB
+ */
+function estimateFileSize(frameRate, resolution, durationMs, mediaType) {
```

Comment:
Neat! Have you done verification to see how actual results compare to the estimate?

I think this can be applied to WEMB without too much trouble if it's also good for gif, right?

Reply:
Not very accurate! This was meant to be a naive implementation based on a formula that's commonly given as a math problem. The resource I linked in the TODO could be summed up as "it isn't easy". I haven't read the full thing yet but I don't think an answer was reached from my skimming

Reply:
Here are some numbers. All tests were done with Orbit 90 at the default view.

Resolution | Estimate | Actual:
540p | 8MB | 6.5MB
720p | 15MB | 10.5MB
1080p | 35MB | 20.9MB

This could possibly be improved upon if I could get the size of the first frame at the resolution specified

Reply:
Oh ok. These numbers are for GIF? Can you look at WEBM too?

Reply:
yes they're for GIF. I think it'll be similar accuracy with WEBM as I'm guessing bit depth and compression ratio. I don't see a resolution in the [webm encoder](https://github.com/uber/hubble.gl/blob/6c044c423c43d37c62fd396fdd6f2a96c885a66a/modules/core/src/encoders/video/webm-encoder.js) though. How is that determined?

Reply:
The resolution is the canvas resolution, which is currently set at a high level from "adapter.scene"

Reply:
is this something that you want to refactor so that it's consistent with the GIF Encoder? Or maybe the other way around where the GIF Encoder should be refactored to match WEBM?

Reply:
They are really separate features. One is the size of the canvas (scene) and the other (gif) is a post-process scaled size.

Reply:
> Can we move the two util functions to a utils.js in the export-video folder?

Will do

Reply:
I see. Is it alright if I do the WEBM size estimation in a different PR then? It sounds like my estimates for GIF and WEBM could be improved since currently one upscales the canvas (WEBM) and the other the final size (GIF?)

Reply:
It's fine to do in another PR. It isn't upscaling though. More accurate to say GIF is downscaling and all other formats just are the set size.

Currently our canvas isn't the right size (too small in the modal) - it's something that'll need to be changed creatively.

Reply:
I see. That's something I can add to the docket on things to discuss